app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params)
```

Comment:
Does this need to create post.comment or can it create a comment independently? Doesn't feel like it should be the posts responsibility to create. I could be wrong, maybe ask Kay?